core/src/main/java/org/wso2/testgrid/core/command/GenerateReportCommand.java

```diff
@@ -74,48 +62,37 @@ public void execute() throws CommandExecutionException {
             logger.info("Generating test result report...");
             logger.info(
                     "Input Arguments: \n" +
-                    "\tProduct name: " + productName + "\n" +
-                    "\tProduct version: " + productVersion + "\n" +
-                    "\tChannel" + channel);
+                    "\tProduct name: " + productName);
 
-            Product product = getProduct(productName, productVersion, channel);
+            Product product = getProduct(productName);
             TestReportEngine testReportEngine = new TestReportEngine();
             testReportEngine.generateReport(product, showSuccess, groupBy);
         } catch (ReportingException e) {
             throw new CommandExecutionException(StringUtil
-                    .concatStrings("Error occurred when generating test report for { product: ", productName,
-                            ", version: ", productVersion, ", channel: ", channel, " }"), e);
+                    .concatStrings("Error occurred when generating test report for { product: ", productName, " }"), e);
         }
     }
 
     /**
      * Returns an instance of {@link Product} for the given product name and product version.
      *
      * @param productName    product name
-     * @param productVersion product version
-     * @param channel        product test plan channel
      * @return an instance of {@link Product} for the given product name and product version
      * @throws CommandExecutionException throw when error on obtaining product for the given product name and product
      *                                   version
      */
-    private Product getProduct(String productName, String productVersion, String channel)
+    private Product getProduct(String productName)
             throws CommandExecutionException {
         try {
             ProductUOW productUOW = new ProductUOW();
-            Product.Channel productChannel = Product.Channel.valueOf(channel);
-            return productUOW.getProduct(productName, productVersion, productChannel)
+            return productUOW.getProduct(productName)
                     .orElseThrow(() -> new CommandExecutionException(StringUtil
                             .concatStrings("No product test plan found for product {product name: ", productName,
-                                    ", product version: ", productVersion, ", channel: ", channel,
+                                    ", product version: ",
```

Comment:
Do we need the ` ", product version: ",` here. as now it is not available to us?